linux_lib/libcrystalhd/libcrystalhd_parser.cpp

```diff
@@ -657,8 +654,8 @@ BC_STATUS DtsAddH264SCode(HANDLE hDevice, uint8_t **ppBuffer, uint32_t *pUlDataS
 				//Succeeded
 
 				//Check NAL Unit Type
-				pNALU = pStart + Ctx->VidParams.StartCodeSz;
-				ulNalSize = Ctx->PESConvParams.m_lStartCodeDataSize;
+				// pNALU = pStart + Ctx->VidParams.StartCodeSz; // uint8_t *pNALU = NULL;
+			        // ulNalSize = Ctx->PESConvParams.m_lStartCodeDataSize; // uint32_t ulNalSize = 0;
```

Comment:
I have updated the branch master on my side; The changes are rebased and separated.

I let you check.